README.md

```diff
@@ -36,3 +36,35 @@ To run the project locally, you can use the `docker-compose.yml` file as such:
 ```
 $ docker-compose up -d
 ```
+
+## Slack App setup
+
+You should first create [a new app](https://api.slack.com/authentication/basics).
+
+Give this app the following scopes:
+
+- `chat:write`
+- `channels:read`
+- `commands`
```

Comment:
These are the only ones I needed to make it work, not sure what else is needed tbh, I'm only using the polling functionality though.